server/plugin/api.go

```diff
@@ -606,6 +607,34 @@ func (p *Plugin) getCommentsForRecord(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (p *Plugin) addCommentsOnRecord(w http.ResponseWriter, r *http.Request) {
+	pathParams := mux.Vars(r)
+	recordType := pathParams[constants.PathParamRecordType]
+	if !constants.ValidRecordTypesForSearching[recordType] {
+		p.API.LogError("Invalid record type while trying to get the record", "Record type", recordType)
+		p.handleAPIError(w, &serializer.APIErrorResponse{StatusCode: http.StatusBadRequest, Message: constants.ErrorInvalidRecordType})
+		return
+	}
+
+	payload, err := serializer.ServiceNowCommentPayloadFromJSON(r.Body)
+	if err != nil {
+		p.API.LogError("Error in unmarshalling the request body", "Error", err.Error())
```

Comment:
```suggestion
		p.API.LogError("error in unmarshalling the request body", "Error", err.Error())
```
I think messages that we are logging on the server start with a small letter and which are passed to UI only start with a capital

Reply:
No @avas27JTG , the logging messages also start with a capital letter. You can check in the whole repo.

Reply:
Okay, then I think I wasn't aware of it, and I was thinking we add small on logs and capital in UI
can you please post the rule that we follow for small/capital messages on MM so that every one of us can read that when needed?

Reply:
Sure, I'll post that.